doc/get-help/help-diagram.rst

```diff
@@ -58,28 +58,14 @@ We look forward to hearing from you!
         pr <-->|my PR is quiet| dev
         contrib -->pr
 
-        classDef quesNodefmt fill:#9DEEF4,stroke:#206C89
+        classDef quesNodefmt font-size:18pt,fill:#9DEEF4,stroke:#206C89
 
-        classDef ansNodefmt fill:#FFAA05,stroke:#E37F17
+        classDef ansNodefmt font-size:14pt,fill:#FFAA05,stroke:#E37F17
 
         classDef boxfmt fill:#FFF5ED,stroke:#E37F17
         class bugcontrib boxfmt
 
-        linkStyle default font-size:20pt,color:#206C89
-
-Flowchart links
```

Comment:
I'd keep the links shown here as a list in addition to the ones now in the diagram, for better accessibility (the font-size in the diagram is pretty small).

Reply:
done in [33d1ff9](https://github.com/pydata/xarray/pull/10324/commits/33d1ff98c5ca5a1a7b65b30c43d562582a731bb4)